connector/connect/common/src/main/scala/org/apache/spark/sql/connect/client/SparkConnectClientParser.scala

```diff
@@ -29,17 +29,21 @@ private[sql] object SparkConnectClientParser {
    *   usage string.
    */
   def usage(): String =
+    // scalastyle:off line.size.limit
     s"""
        |Options:
        |   --remote REMOTE          URI of the Spark Connect Server to connect to.
        |   --host HOST              Host where the Spark Connect Server is running.
        |   --port PORT              Port where the Spark Connect Server is running.
-       |   --enable-ssl             Connect to the server using SSL.
+       |   --use_ssl                Connect to the server using SSL.
        |   --token TOKEN            Token to use for authentication.
        |   --user_id USER_ID        Id of the user connecting.
        |   --user_name USER_NAME    Name of the user connecting.
+       |   --user_agent USER_AGENT  The User-Agent Client information (only intended for logging purposes by the server).
```

Comment:
An explanation of `user_agent`
https://github.com/apache/spark/blob/c7795bb8cc82073ba555aaa233e3b2586ae6d1eb/connector/connect/common/src/main/scala/org/apache/spark/sql/connect/client/SparkConnectClient.scala#L676

https://github.com/apache/spark/blob/c7795bb8cc82073ba555aaa233e3b2586ae6d1eb/connector/connect/common/src/main/protobuf/spark/connect/base.proto#L72-L75